metrics-core/src/traits.rs

```diff
@@ -51,9 +63,16 @@ pub trait Gauge: MetricCommon {
 
 /// Any sort of summary of the record values
 pub trait Summary: MetricCommon {
+    // TODO: function parameters for record are inconsistent with other traits
```

Comment:
This is because adding a count parameter doesn't make sense for counters or gauges but does for a summary. (IMO at least). Storing a value into a counter 30 times is no different from storing it once.

Not sure how we'd make this consistent with the other traits. (Unless you're talking about parameter naming here which can definitely be changed)

Reply:
Just about the parameter naming